app/views/service_requests/new.js.erb

```diff
@@ -0,0 +1,9 @@
+<% if @service_request.attributes[:location_id].kind_of?(Array) %>
```

Comment:
Why we can't use simple_form errors? Why errors stored in `@service_request.data[:location_id]`?